testutils/infra/container_manager/docker_compose_manager.py

```diff
@@ -0,0 +1,136 @@
+
+import os
+import time
+import subprocess
+import filelock
+import logging
+import tempfile
+
+from . import log_files
+from .base import BaseContainerManagerNamespace
+
+# Temporary hack: assume the caller sets the variable to whatever is
+# used globally (ie.e conftest.docker_compose_instance)
+# This will eventually be the id of the namespace
+docker_compose_instance = "invalid"
+
+# Global lock to sycronize calls to docker-compose
+docker_lock = filelock.FileLock("docker_lock")
+
+def get_host_ip():
+    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
```

Comment:
Not sure if you have strict requirements here, but I would use:
`return socket.gethostbyname(socket.gethostname())`

Reply:
I tried, but `socket.gethostbyname(socket.gethostname())` returns the loopback address, 127.0.1.1 in my machine, while the original implementation returns the LAN address. The test does not seem to work if setting the docker environment with the loopback address.

Reply:
Disregard my previous comment: it was failing locally due to a different bug in EnterpriseSetup(s). Indeed your suggested change works and it is way simpler, see commit https://github.com/mendersoftware/integration/pull/764/commits/5b1b24693f37cd264905a11f8f49ac33567f6d90